src/emmy/expression/render.cljc

```diff
@@ -6,25 +6,12 @@
   (:require [clojure.set :as set]
             [clojure.string :as s]
             [clojure.zip :as z]
-            [emmy.expression :as x]
-            [emmy.expression.cse :refer [extract-common-subexpressions]]
+            [emmy.generic :as g]
             [emmy.pattern.rule :as R :refer [=>]]
             [emmy.ratio :as r]
+            [emmy.simplify.rules :refer [negative-number?]]
             [emmy.util :as u]
-            [emmy.value :as v]
-            [emmy.util.permute :as p]))
-
-(defn- make-symbol-generator [p]
-  (let [i (atom 0)]
-    (fn [] (symbol
-           #?(:clj
-              (format "%s%04x" p (swap! i inc))
-
-              :cljs
-              (let [suffix (-> (swap! i inc)
-                               (.toString 16)
-                               (.padStart 4 "0"))]
-                (str p suffix)))))))
+            [emmy.value :as v]))
 
 (def ^{:private true
```

Comment:
once we get this merged I am going to move the docstrings down below the symbol, like in #110 . I seriously thought that was not a thing, leading to the 100s of `:doc` forms... whoops!